### ISSUE TITLE
Reactive data example not working

### ISSUE THREAD

**Issue by kjfdebruin** [apertureless/vue-chartjs#122]
### Expected Behavior
Reactive LineChart example from the [docs ](http://vue-chartjs.org/#/home?id=reactive-data) should render the chart on load, and re-render the chart on button click

### Actual Behavior
The chart does not render (axes are drawn, but chart area stays blank), even though chartData is populated correctly (as seen with VueJS Chrome plugin).
![image](https://user-images.githubusercontent.com/10156093/27282810-ecf44734-54f1-11e7-8dde-ab68e1f7008e.png)

On click I get:

> [Vue warn]: Error in callback for watcher "chartData": "TypeError: Cannot set property 'label' of undefined"

and 

> TypeError: Cannot set property 'label' of undefined

### Environment
- vue.js version: 2.3.4
- vue-chart.js version: 2.6.5
- npm version: 4.2.0

**Comment by apertureless** [comment]
Can you please provide a minimal reproduction? Codepen or jsfiddle?

**Comment by kjfdebruin** [comment]
Struggled a bit getting this into one JS file. Not sure if this is a true reproduction. The graph doesn't render, but I'm not getting any console errors when clicking on the button.

Here is my [fiddle](https://jsfiddle.net/kareldb/wyfz58jL)

I worked from this example [fiddle](https://jsfiddle.net/yomi/k61epfkq/)

**Comment by apertureless** [comment]
You have to pass the mixin as an array ;)

```javascript
mixins: [VueChartJs.mixins.reactiveProp]
```

here is also a working codepen https://codepen.io/apertureless/pen/BZKBdX

**Comment by kjfdebruin** [comment]
Thanks so much for checking this out and providing the codepen.

I had the mixin as an array in my code, just not in the fiddle (sorry about that), so that wasn't the issue. Working from your codepen I realized that the issue was that the datacollection variable was initialized as `null.`

Would you mind confirming that this is the issue? Reactive data cannot be initialized as null.

**Comment by apertureless** [comment]
Hey @kjfdebruin 

that seem to be the problem. However it actually should work with an empty initial data model. Could be related to https://github.com/apertureless/vue-chartjs/pull/109

Because as far as I remember, it worked with an empty / null data model before. 

I will digg into it and fix it ASAP. However a temporary fix would be to initialize your data model as an empty object.

```javascript
data () {
  return {
    datacollection: {}
  }
}
```

**Comment by kjfdebruin** [comment]
Thanks!

Yes, it works when initializing as an empty object. 

What is interesting, is that with
`responsive: true, maintainAspectRatio: false`
then **with a data model** e.g.
`message: {
                      labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
                      datasets: [
                        {
                          label: 'Data One',
                          backgroundColor: '#f87979',
                          data: [40, 39, 10, 40, 39, 80, 40]
                        }
                      ]
                    }`
the height looks like this:
![image](https://user-images.githubusercontent.com/10156093/27324379-52e0dae0-55a5-11e7-8c7d-584c5b202469.png)
**with an empty data model** e.g.
`message: {}`
it looks like this:
![image](https://user-images.githubusercontent.com/10156093/27324483-a5307080-55a5-11e7-818c-745b6f294ea8.png)

(Note the different height)

**Comment by apertureless** [comment]
Hey, this issue is fixed with #135 and will be available with the next release `2.7.0`

**Comment by iamanas20** [comment]
Hello, I had this issue and initializing the chart-data like @apertureless said was not enough, as I had this error:
```
[Vue warn]: Error in callback for watcher "chartData": "TypeError: Cannot read property 'map' of undefined"
```
I fixed it by initializing the chart-data like the following:

```
thirty_chartdata: {
  labels: [''],
  datasets: []
},
```
Empty labels and empty datasets removed that error for me, anyone had the same thing?
